BigQuery/13_iv_antibiotics_pharmacy.sql

```diff
@@ -0,0 +1,86 @@
+CREATE OR REPLACE TABLE `physionet.iv_abx_pharmacy` as
+
+select 
+subject_id,
+hadm_id,
+stay_id,
+DATETIME_TRUNC(starttime, HOUR) as chart_hour,
+intime,
+outtime,
+medication,
+-- todo: instead of taking the arbitrary minimum, take the values from the latest entertime for a given stay_id/chart_hour/medication combination 
+min(entertime) as entertime_min,
+min(starttime) as starttime_min,
+min(route) as route_min,
+max(stoptime) as stoptime_max,
+min(frequency) as frequency_min,
+min(disp_sched) as disp_sched_min,
+min(doses_per_24_hrs) as doses_per_24_hrs_min,
+min(duration) as duration_min,
+min(duration_interval) as duration_interval_min,
+count(1) as records
+from `physionet-data.mimic_hosp.pharmacy`
+JOIN `physionet-data.mimic_icu.icustays`
+USING (subject_id, hadm_id)
```

Comment:
I know, I'm repeating myself here, but I wanted to point out the repetition of using the subject id and hadm id together when only one _might_ be needed.